docs/setup/kubeedge_install.md

```diff
@@ -0,0 +1,76 @@
+# Install KubeEdge
+
+In this section, we would cover the below topics
+
+1. [Abstract](#Abstract)
+2. [Pre-Requisite](#pre-requisite)
+3. [Setup KubeEdge](#setup-kubeEdge)
+4. [Configure KubeEdge](#configure-kubeEdge)
+5. [Run KubeEdge](#run-kubeedge)
+6. [KubeEdge Pre-Check](#kubeedge-pre-check)
+## Abstract
+KubeEdge is composed of cloud and edge sides. It is built upon Kubernetes and provides core infrastructure support for networking, application deployment and metadata synchronization between cloud and edge. So if we want to setup KubeEdge, we need to setup Kubernetes cluster, cloud side and edge side.
```

Comment:
adding the usage of an already existent cluster

Reply:
@subpathdev I don't understand this one. What do you mean?

Reply:
You don't have to create a new cluster you can use an existing one. reading this, feels like I have to create a new cluster.

Reply:
agreed, please update the content